vpr/src/place/place.cpp

```diff
@@ -732,7 +737,9 @@ void try_place(const t_placer_opts& placer_opts,
 
     first_rlim = (float)max(device_ctx.grid.width() - 1, device_ctx.grid.height() - 1);
 
-    float first_t = starting_t(&costs, &prev_inverse_costs,
+    float first_t = 0;
```

Comment:
This needs a big comment on how the analytic placer and annealer interact.  Analytic placer: disables most of the anneal (leaves the quench) and passes its placement back to the quench via locations marked on both the clb_netlist and the grid.